### PR TITLE
Login screen UI makeover

### PR DESCRIPTION
**Pull request by anya-ji**

### CODE REVIEW

File: ui/AssessmentManager.py [hunk 1]
```diff
@@ -31,9 +31,9 @@ def finished_init(self, *args):
         if len(ids) > 2:
             self.user = ids.user
             self.module_number = ids.module_number
-            self.app.title = "Health Friend [Assessment]  ::  " + self.user['first_name'] + " " + self.user['last_name']
-        else:
-            self.app.title = "Health Friend [Assessment]  ::  EWH"
+        #     self.app.title = "Health Friend [Assessment]  ::  " + self.user['first_name'] + " " + self.user['last_name']
+        # else:
+        #     self.app.title = "Health Friend [Assessment]  ::  EWH"
```
Comment (navinramsaroop) [inline ui/AssessmentManager.py:36]: Can we delete these lines

**Comment by anya-ji** [comment]
Screen Shot 2020-10-06 at 12 40 29

File: ui/AssessmentManager.py (hunk 1 above)
Comment (anya-ji) [reply ui/AssessmentManager.py]: I uncommented it for now, since it only appears on the window's title bar anyways

**Review by tklausner — APPROVED** [review]
Looks good to me.

**Pull request merged**